app/services/complete_moab_handler.rb

```diff
@@ -186,6 +186,10 @@ def moab_validator
     @moab_validator ||= MoabValidator.new(druid: druid, storage_location: storage_location, results: results)
   end
 
+  def checking_primary_moab?
```

Comment:
would this be better named `is_primary_moab?`  the method name is throwing me, because I don't think of "checking" being an appropriate verb for updates.   Other ideas: `moab_is_primary` ?  `handling_primary_moab`?  (this is the complete_moab_HANDLER) `primary_moab(complete_moab)` ?  Or roll your own.

Reply:
How about a `.primary?` method in the model itself?   so `complete_moab.primary?` and in complete_moab_handler, that could be memoized ...

Reply:
@ndushay, done, thanks for the suggestion!  mind having another look when you get a chance?